MAUI/ComboBox/Editing.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Editing in .NET MAUI ComboBox control | Syncfusion
+description: Learn here all about Editable modes support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more.
+platform: maui
+control: SfComboBox
+documentation: ug
+---
+
+# Editing in .NET MAUI ComboBox (SfComboBox)
+
+The `ComboBox` control supports both editable and non-editable text boxes for selecting an item from a data source. To enable editing functionality, set `IsEditable` property as `true`. The default value is `false`.
+
+## Editable ComboBox
+
+In editable mode, the `ComboBox` allows users to edit in the text box and it automatically appends the remaining letters to the entered text when it is valid.
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfComboBox x:Name="comboBox"
+                    WidthRequest="250"
+                    IsEditable="true"
+                    ItemsSource="{Binding SocialMedias}"
+                    DisplayMemberPath="Name"
+                    TextMemberPath="Name">
+</editors:SfComboBox>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+comboBox.IsEditable = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox choose item using editing](Images/Editing/EditableMode.png)
+
+## Non-editable ComboBox
+
+Non-editable mode prevents users from editing and instead allows them to select from drop-down list.
```

Comment:
Non-editable mode prevents users from editing and instead allows them to select from the drop-down list.